slc/component/matter-examples/matter_zigbee_concurrent.slcc

```diff
@@ -15,5 +15,7 @@ requires:
 define:
   - name: SL_MATTER_ZIGBEE_CMP
     value: "1"
+  - name: SL_RAIL_UTIL_IEEE802154_FAST_CHANNEL_SWITCHING_DEFAULT_ENABLED
```

Comment:
Perhaps this should go into matter_zigbee_multiprotocol_common.slcc -- thats where fast channel switching is enabled for all boards for all flavors of the CMP app (concurrent and sequential).

Reply:
Actually, I think the change should go both here and there.

Reply:
Merged your PR.